app/src/main/java/com/zulip/android/widget/ZulipRemoteViewsFactory.java

```diff
@@ -34,32 +37,35 @@ public void onCreate() {
 
     }
 
-    private String setupWhere() {
+    private Calendar getMinDate() {
+        Calendar calendar = Calendar.getInstance();
         switch (from) {
             //These values present in R.arrays.from_values
             case "today":
-                return "timestamp BETWEEN DATE('now') AND DATE('now', '+1 day')";
+                break;

```

Comment:
The Calendar.getInstance() method returns a Calendar object with the current date, so in case the minimum date is today, there is no need to modify it.